service/routes_test.go

```diff
@@ -1,69 +0,0 @@
-package service_test
```

Comment:
personally quite down to remove the unit tests, I feel that a lot of them are bloat... but big deletions like these should get eyes from the wider team also